spec/serializers/disability_claim_detail_serializer_spec.rb

```diff
@@ -64,4 +64,15 @@
       expect(other_documents.select { |obj| !obj['tracked_item_id'].nil? }.count).to eq 0
     end
   end
+
+  context 'with some phase dates' do
+    it 'should not have a phase 1..6 event' do
+      (1..6).each do |i|
+        expect(attributes['events_timeline'].select { |obj| obj['type'] == "phase#{i}" }.count).to eq 0
+      end
+    end
+    it 'should have a phase 7 event' do
+      expect(attributes['events_timeline'].select { |obj| obj['type'] == 'phase7' }.count).to eq 1
```

Comment:
Can this also test that the date was parsed correctly? That functionality was also moved in the refactor